packages/aws-cdk-lib/aws-appconfig/README.md

```diff
@@ -78,6 +78,24 @@ new appconfig.Environment(this, 'MyEnvironment', {
 Environment monitors also support L1 `CfnEnvironment.MonitorsProperty` constructs through the `fromCfnMonitorsProperty` method.
 However, this is not the recommended approach for CloudWatch alarms because a role will not be auto-generated if not provided.
 
+See [About the AWS AppConfig data plane service](https://docs.aws.amazon.com/appconfig/latest/userguide/about-data-plane.html) more information.
```

Comment:
*for more information

Reply:
@daschaa 
Thank you for your review.
I added `for`.